jac-uk/admin
# Upgrade to use node 16

## What's included?
Upgrade to use node v16.15.1
Fixes bug creating preview URLs
Fixes bug preventing deployment

NOTE: we will fix dependency issues separately

## Who should test?
✅ Developers

## How to test?
Using node v16.15.1 developers should be able to successfully run the following commands:
 - `npm ci`
 - `npm run lint`
 - `npm run test`
 - `npm run serve`
 - `npm run build`

The following workflows should work as expected:
 - this PR should have a preview URL automatically created and updated as well as lint and tests run
 - merging this PR should deploy to develop and staging
 - publishing a release on `main`, once this PR has been merged, should deploy to production
 - running the 'Deploy to Production' workflow should deploy to production

## Risk - how likely is this to impact other areas?
🟠 Medium risk - this effects the creation of preview urls and the build pipeline

---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_
